### PR TITLE
minor changes

### PR DESCRIPTION
**Pull request by proddy**
- show scheduler name in log, helps troubleshooting
- tidy up error messages so its more consistent
- more refactoring of how commands are managed. all tests passed so think its ok
- update tests for HC1, HC2
- added bbqkees icon to Hardware Status in Web
- added EMS-ESP version as a HA sensor (https://github.com/tp1de/ioBroker.ems-esp/issues/84)

### CODE REVIEW

File: src/devices/thermostat.cpp [hunk 1]
```diff
@@ -269,8 +269,10 @@ std::shared_ptr<Thermostat::HeatingCircuit> Thermostat::heating_circuit(std::sha
                 toggle_ = true;
                 break;
             }
+            Serial.println();
```
Comment (MichaelDvP) [inline src/devices/thermostat.cpp:272]: What is ths used for?

File: src/emsdevice.cpp [hunk 2]
```diff
@@ -332,7 +332,7 @@ void EMSdevice::fetch_values() {
 // toggle on/off automatic fetch for a telegramID
 void EMSdevice::toggle_fetch(uint16_t telegram_id, bool toggle) {
 #if defined(EMSESP_DEBUG)
-    EMSESP::logger().debug("Toggling fetch for deviceID 0x%02X, telegramID 0x%02X to %d", device_id(), telegram_id, toggle);
+    EMSESP::logger().debug("Setting fetch to %d for deviceID 0x%02X, telegramID 0x%02X to %d", toggle, device_id(), telegram_id);
```
Comment (MichaelDvP) [inline src/emsdevice.cpp:335]: referenced 4 values, but only 3 listed.

File: src/emsesp.cpp [hunk 3]
```diff
@@ -746,11 +746,19 @@ void EMSESP::publish_response(std::shared_ptr<const Telegram> telegram) {
...
+            found_device = true;
+            if (emsdevice->get_value_info(root, cmd, id)) {
+                return true;
+            }
         }
     }
+    // if the EMS device was valid, but the cmd not found show an error
+    if (found_device) {
+        // return EMSESP::return_not_found(root, "poep", cmd); // not found
+    }
```
Comment (MichaelDvP) [inline src/emsesp.cpp:761]: "poep"?

**Review by MichaelDvP — APPROVED** [review]

File: src/command.cpp [hunk 4]
```diff
@@ -523,7 +529,13 @@ void Command::show(uuid::console::Shell & shell, uint8_t device_type, bool verbo
     std::list<std::string> sorted_cmds;
     for (const auto & cf : cmdfunctions_) {
         if ((cf.device_type_ == device_type) && !cf.has_flags(CommandFlag::HIDDEN)) {
-            sorted_cmds.push_back(tagged_cmd(cf.cmd_, cf.flags_));
+            // remove duplicates
+            auto line = tagged_cmd(cf.cmd_, cf.flags_);
+            // only add if not already in list
+            // removes duplicates if having more than one device (e.g. 2 thermostats)
```
Comment (MichaelDvP) [inline src/command.cpp:535]: This is not needed. We don't have dublicates, they always differ in heatingcircuit.

File: src/emsesp.cpp (hunk 3 above)
Comment (proddy) [reply src/emsesp.cpp:761]: :-) how I was feeling last night, will remove

File: src/command.cpp (hunk 4 above)
Comment (proddy) [reply src/command.cpp]: yes you're right - i will need to find why in the tests when I add to thermostats with two different HCs its not working (the test is called 2thermostats)

**Comment by proddy** [comment]
thanks for the quick review and sorry for the messy PR. I made some changes.

File: src/command.cpp (hunk 4 above)
Comment (proddy) [reply src/command.cpp]: and I assume having multiple solar, mixers etc won't happen either? just to double check

File: src/system.cpp [hunk 5]
```diff
@@ -1414,7 +1413,7 @@ bool System::command_info(const char * value, const int8_t id, JsonObject output
 #endif
     EMSESP::esp8266React.getNetworkSettingsService()->read([&](NetworkSettings & settings) {
         if (WiFi.status() == WL_CONNECTED && !settings.bssid.isEmpty()) {
-            node["BSSID"] = "set"; // TODO why is this not the actual value?
+            node["BSSID"] = "set"; // TODO why is this not the actual value??
```
Comment (MichaelDvP) [inline src/system.cpp:1416]: A while ago a user complains tht he does not want to publish his mac-address in the support info. So you have commented out all macs and ips and here we only see if a bssid is set or not.

**Review by MichaelDvP — APPROVED** [review]

File: src/system.cpp (hunk 5 above)
Comment (proddy) [reply src/system.cpp:1416]: ok.

**Pull request merged**